save-backend/src/test/kotlin/org/cqfn/save/backend/controller/ExecutionControllerTest.kt

```diff
@@ -114,6 +116,8 @@ class ExecutionControllerTest {
             Sdk.Default.toString(),
             null,
             null,
+            null,
+            null,
```

Comment:
I tihnk it's time to create `Execution.stub()` simliarly to `Project`

Reply:
Added